### ISSUE TITLE
Problem: Using Mac Mini to build darwin packages

### ISSUE THREAD

**Issue by clacke** [fractalide/fractalide-infra#4]
There is a package called [darling](/darlinghq/darling), which implements XNU kernel calls, a Mach-O loader and other things necessary for making Linux run macOS binaries.

It would be great if this could be used for darwin crossbuilds directly on NixOS instead of relying on a weaker machine through a slow network connection.

https://wiki.darlinghq.org/build_instructions

**Comment by sjmackenzie** [comment]
yeah now that build times have reduced greatly, probably best to focus on the wallet.

**Comment by clacke** [comment]
Oh yeah, this wouldn't affect code review interactivity or anything like that, it's just that our use of macOS irks me, and just running darling on the beefy main machine would reduce moving parts and improve performance on hydra side.

And this might be a better alternative than trying to create a new PureDarwin Pico to run Nix on, which has also been on my mind. :-)

Super low priority. I just heard about darling and had to write this down somewhere.
